packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -1,21 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { string, number, shape } from 'prop-types';
 import { Link, resourceUrl } from '@magento/venia-drivers';
 import { Price } from '@magento/peregrine';
-
-import classify from '../../classify';
 import { transparentPlaceholder } from '@magento/peregrine/lib/util/images';
-import { generateSrcset } from '../../util/images';
+
+import { mergeClasses } from '../../classify';
+import Image from '../Image';
 import defaultClasses from './item.css';
 
 // The placeholder image is 4:5, so we should make sure to size our product
 // images appropriately.
 const imageWidth = '300';
 const imageHeight = '375';
 
-const ItemPlaceholder = ({ children, classes }) => (
+const ItemPlaceholder = ({ classes }) => (
     <div className={classes.root_pending}>
-        <div className={classes.images_pending}>{children}</div>
+        <div className={classes.images_pending}>
+            <Image
+                alt="Placeholder for gallery item image"
+                classes={{ root: classes.image_pending }}
+                src={transparentPlaceholder}
```

Comment:
Something is still wrong with placeholder images in gallery: 

[![Image from Gyazo](https://i.gyazo.com/f286c69debc68ce5dfed2c2bdab8c05f.gif)](https://gyazo.com/f286c69debc68ce5dfed2c2bdab8c05f)

Reply:
👍 , I'll take a look.

Reply:
I added the concept of "layout only" placeholder styles to `Image` for this. See `classes.placeholder_layoutOnly`.

These styles _hold the place_ in the page while the image loads. If `usePlaceholder` is true (the default), it will also draw a gray box there.

https://github.com/magento/pwa-studio/pull/1871/commits/50761a370eaf396a6eec1e489e3766b2f430e096